tests/init.sh

```diff
@@ -1,8 +1,15 @@
-source common.sh
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+# Don't start the daemon
+source common/vars-and-functions.sh
```

Comment:
@thufschmitt to answer the question you asked on IRC a bit better, I didn't put the starting the daemon in init for these reasons:

1. Running tests without recalling init should still work. if one wants to examine state: the test itself starts and stops the daemon, rather than something else starting and it stopping.
2. `init.sh` now stops any rougue daemon, and never starts any. It is just to reset the state. 

The file is split because `init.sh` still needs the env vars.

Reply:
Oh yes of course, it’s nice to be able to manually run `init.sh` once and then the test (I actually do that all the time)